### PR TITLE
make internal SARAH models loadable on parallel kernels

### PR DESCRIPTION
**Pull request by wkotlarski**

### CODE REVIEW

File: meta/FlexibleSUSY.m [hunk 1]
```diff
@@ -2057,8 +2057,13 @@ corresponding tadpole is real or imaginary (only in models with CP
...
                     (* don't pollute terminal with SARAH initialization message *)
-                    Block[{Print},
+                    Block[{Print,workingDirectory = Directory[]},
                        << SARAH`;
+                       SARAH`SARAH[OutputDirectory] = FileNameJoin[{workingDirectory, "Output"}];
+                       SARAH`SARAH[InputDirectories] = {
+                          FileNameJoin[{workingDirectory, "sarah"}],
+                          ToFileName[{$sarahDir, "Models"}]
+                       };
                        Start@modelName;
```
Comment (Expander) [inline meta/FlexibleSUSY.m:2067]: One question regarding `Start@modelName;`

SARAH also has so-called "sub-models". An example is the `MSSM/CKM` or `MSSM/CPV`. To my knowledge these models are loaded as:
~~~.m
Start["MSSM", "CPV"];
~~~
I'm wondering if the above call of `Start` works also with sub-models. Did you test this at some point?

Comment (wkotlarski) [reply meta/FlexibleSUSY.m:2067]: In the `MSSMCPV` `modelName` is `MSSM/CPV`.  The old syntax `Start[MSSM/CPV]` still works so there is no problem there, at least for now.

**Comment by wkotlarski** [comment]
I'm seeing a problem with models which are not in `FlexibleSUSY/sarah`. For example, in the `MSSM` we not get a warning
```
From remote kernel:
FileByteCount::fdnfnd: 
   Directory or file "/tmp/FS/sarah/MSSM/MSSM.m" not found.
```

File: meta/FlexibleSUSY.m (hunk 1 above)
Comment (Expander) [reply meta/FlexibleSUSY.m:2067]: Ah, ok, thanks for checking!

**Comment by Expander** [comment]
Hmm, could it be that the variable `$sarahDir` is not distributed to the kernel?

**Comment by wkotlarski** [comment]
Maybe, but this problem has nothing to do with parallelization or with anything I'm doing right now. It seems to have always existed in FS. This
```
Current working directory: /tmp/FS
SARAH output directory: /tmp/FS/Output
Starting model: Start[MSSM]
Preparing arrays
 ... checking Directory: /tmp/FS/sarah

FileByteCount::fdnfnd: 
   Directory or file "/tmp/FS/sarah/MSSM/MSSM.m" not found.
 ... checking Directory: /home/wojciech/HEP-software/mathematica/SARAH-4.15.1\
 
>    /Models/
```
comes from a "normal" FS run.

**Comment by wkotlarski** [comment]
I think that it's because of this order
```mathematica
SARAH`SARAH[InputDirectories] = {
   FileNameJoin[{workingDirectory, "sarah"}],
   ToFileName[{$sarahDir, "Models"}]
};
```
Those two dirs are checked in order. So depending on whether the model is in `sarah` or `Models` you will always get this warning for one of the types.

**Comment by Expander** [comment]
Ah, of course! Personally I've never seen this warning before.

But the behaviour to first check in FS's `sarah/` directory and then in SARAH's `Models/` directory is what we want.

**Review by Expander — APPROVED** [review]

**Pull request merged**